examples/astro/src/pages/api/demo-workflow.ts

```diff
@@ -0,0 +1,17 @@
+import { serve } from "@upstash/workflow/astro";
+
+export const { POST } = serve(async (ctx, workflow) => {
+  // Get all links were accessed before 7 days from now or not access at all
+  const results = await workflow.run("step-1", async () => {
+    console.log(ctx.request.url);
+    return [];
+  });
+
+  if (!results.length) {
+    return;
+  }
+
+  await workflow.run("step-2", async () => {
+    // Do step 2
+  });
+});
```

Comment:
```suggestion
export const { POST } = serve(async (ctx, workflow) => {

  const env = workflow.env;

  const results = await workflow.run("step-1", async () => {
    console.log(ctx.request.url);
    return [];
  });

  if (!results.length) {
    return;
  }

  await workflow.run("step-2", async () => {
    // Do step 2
  });
}, {
  env: {
    QSTASH_TOKEN: import.meta.env.QSTASH_TOKEN
  }
});
```

Since vite serves the environment variables with `import.meta.env.*`, SDK can't access it. Therefore we need to explicitly pass the variable as in example